toolchains/rbe_toolchains_config.bzl

```diff
@@ -96,7 +96,7 @@ def _envoy_rbe_toolchain(name, env, toolchain_config_spec_name, toolchain_config
     )
 
 def rbe_toolchains_config(generator = False, force = False):
-    linux_exec_properties = create_rbe_exec_properties_dict(docker_add_capabilities = "SYS_PTRACE,NET_RAW,NET_ADMIN", docker_network = "standard")
+    linux_exec_properties = create_rbe_exec_properties_dict(docker_add_capabilities = "SYS_PTRACE,NET_RAW,NET_ADMIN", docker_network = "standard", docker_ulimits="memlock=-1,nice=-20,rtprio=10,stack=8192")
```

Comment:
i think you might need to `*1024` - not sure how rbe works but testing locally - eg:

```
$ docker run -ti --rm --ulimit stack=8388608 busybox sh
/ # ulimit -s
8192
```

Reply:
Yes, good catch